### PR TITLE
Include binance example in readme.md

### PR DESCRIPTION
**Pull request by berithpy**
Included binance example for eth on readme.md, this may result in some more traffic from google and also a quickstart for new miners

### CODE REVIEW

**Review by yazanswe — COMMENTED** [review]
gf

**Review by Chockzche — COMMENTED** [review]
Thanks

**Review by Angel1983-ia — APPROVED** [review]

**Review by scotts2080 — APPROVED** [review]

**Review by scotts2080 — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -66,6 +66,7 @@ Be aware when you download NBMiner binaries from other sources.
 - **herominers:** nbminer -a ethash -o ethproxy+tcp://ethereum.herominers.com:10201 -u 0x12343bdgf.worker
 - **nicehash:** nbminer -a ethash -o nicehash+tcp://daggerhashimoto.eu.nicehash.com:3353 -u btc_address.worker
 - **miningpoolhub**: nbminer -a ethash -o nicehash+tcp://asia.ethash-hub.miningpoolhub.com:20535 -u username.worker
+- **binance:** nbminer -a ethash -o ethproxy+tcp://ethash.poolbinance.com:8888 -u account.worker
```
[the remark this answers is not in the document]
Comment (scotts2080) [reply readme.md:69]: bc1qldlx2rrg4jx9cl8zuc39fxjjpa4udyasw8emay

**Review by scotts2080 — APPROVED** [review]
https://developer.trustwallet.com/developer/wallet-core

**Review by scotts2080 — APPROVED** [review]

**Pull request closed without merging**